pkg/core/resources/apis/meshexternalservice/api/v1alpha1/helpers.go

```diff
@@ -73,7 +71,7 @@ func (t *MeshExternalServiceResource) FindPortByName(name string) (core.Port, bo
 }
 
 func (m Match) GetName() string {
-	return fmt.Sprintf("%d", m.Port)
+	return ""
```

Comment:
This seems super odd to me. @lukidzi WDYT?

Reply:
Yes, MES only has 1 port probably that's why

Reply:
```suggestion
        // MES can only have one port so there's no point in naming it.
	return ""
```